x11-misc/screengrab/screengrab-2.3.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg-utils
+
+DESCRIPTION="Qt GUI Screenshot Utility"
+HOMEPAGE="https://lxqt.github.io/"
+SRC_URI="https://github.com/lxqt/${PN}/releases/download/${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2 GPL-2+ LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
+IUSE=""
+
+BDEPEND="dev-qt/linguist-tools:5"
+DEPEND="
```

Comment:
It doesn't look like the dependency is in the link you showed :thinking: 

The executable is otherwise linked against `libxcb-xfixes.so.0` which belongs to `x11-libs/libxcb` and is already a dependency :thinking:

Reply:
So it seems!

I don't remember the process any longer, so my eyes may have been blurry enough to combine the extras package with the fixes from XCB some lines down from there.

There's probably a historical reason for the 'libXfixes' one, mistake or not.

Thanks!

Reply:
Thank you too ! So you want it back to the dependency list or we're good ?